app/assets/stylesheets/ltags/UserSubscriptionTag.scss

```diff
@@ -1,85 +1,74 @@
+@import 'config/import';
+
 .ltag__user-subscription-tag {
-  .ltag__user-subscription-tag__container {
+  &__container {
+    border-radius: var(--radius);
+    border: 2px solid var(--body-color);
+    box-shadow: 4px 4px 0 var(--body-color);
```

Comment:
You should be able to use SCSS generator we have for these boxes:

- this is a function: https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/config/_import.scss#L206-L231 - it’s documented there as well
- and this is an example of how to use it: https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/components/notices.scss#L5-L10

Reply:
Thank you!